sirepo/server.py

```diff
@@ -658,6 +658,9 @@ def init(uwsgi=None, use_reloader=False, is_server=False):
     uri_router.init(_app, simulation_db)
     if is_server:
         sirepo.db_upgrade.do_all()
+        # giving this info to util so it knows to show message
```

Comment:
`# Currently used for a special case in sirepo.util.in_flask_request. Do not use widely, because we should avoid server vs pkcli dependencies.`